functors_task_1/student/include/bar_serving.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include <array>
-
```

Comment:
No, no, please don't!
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file